src/enums/entity-type.ts

```diff
@@ -1,29 +1,31 @@
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+
 /** Enum for Entity Type codes. */
 export enum EntityType {
   // BC Entity Types:
-  BC = 'BC',
-  CC = 'CC',
-  CP = 'CP',
-  CR = 'CR',
-  DBA = 'DBA',
-  FI = 'FI',
-  FR = 'FR',
-  GP = 'GP',
-  LL = 'LL',
-  LP = 'LP',
-  PA = 'PA',
-  PAR = 'PAR',
+  BC = CorpTypeCd.BC_COMPANY,
```

Comment:
This makes things much easier and cleaner for proceeding with the way of navigating work 😁

I think we'll have to change the value of the `supported-incorporation-registration-entities` FF we created. Also, I'm going to make the necessary changes in my two tickets to accompany this adjustment after you merge Sev 👍

Reply:
OK. Let me know about that existing FF, and whether we need another one for something different.

I haven't figured out the build issue yet...